Makefile

```diff
@@ -28,6 +28,8 @@ SMTP_ADDRESS ?= ''
 SMTP_PASS ?= ''
 SMTP_PORT ?= ''
 SMTP_FROM ?= ''
+MAINTENANCE_DAY ?= ''
+MAINTENANCE_HOUR ?= ''
```

Comment:
Minor thing but would not it be better if there are some defaults here? Looking at the code it seems that Thursday 2-3 AM are defaults if nothing is specified so not a big deal, just for the sake of consistency. In OCM these params are required so user will have to select some option anyway.

Out of curiosity, why these defaults, I would expect Sunday? :)

Reply:
Thanks Tomas. I didn't add defaults as none of the other params had them and as you said we have defaults set elsewhere. But if you think it would be better I can add them.
@adam-cattermole might be able to clarify why the default is set to Thursday at 2am?

Reply:
Hey, so these defaults are what have been used since 1.28.0 from what I can tell (from before the GCP branch was merged) and were set as the default once we removed RHMIConfig - https://github.com/integr8ly/integreatly-operator/commit/f6ef2a954e4b4f5a845b62bdf55b70d166e2244e

Reply:
Thanks for all the replies. I have no objections for this staying as is